android/src/main/java/com/onegini/mobile/sdk/reactnative/RNOneginiSdk.kt

```diff
@@ -257,6 +257,14 @@ class RNOneginiSdk(private val reactContext: ReactApplicationContext) : ReactCon
         }
     }
 
+    @ReactMethod
+    fun deregisterAuthenticator(idOneginiAuthenticator: String?, promise: Promise) {
+        when {
+            idOneginiAuthenticator == null -> promise.rejectWithNullError(IdOneginiAuthenticator.paramName, IdOneginiAuthenticator.type)
```

Comment:
why is this `IdOneginiAuthenticator` instead of `OneginiAuthenticatorId` ? 🤔

Reply:
This problem is already picked up in another commit for RNP-109 which builds ontop of RNP-107, https://github.com/onewelcome/sdk-react-native/pull/129/commits/f4b05ba4b5c424debbcdff94e2c294035d060dc8